app/components/sort_links/style.scss

```diff
@@ -40,3 +40,8 @@
 .app-sort-links__item {
   padding: 0.3125em;
 }
+
+.app-sort-links__item--no_padding {
```

Comment:
because the sort links are configured like an unordered list, there was some weird padding going on if the sort options ran onto a second line. So I've removed that padding.